src/factory/index.js

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The idea was that I wanted to make it clear that the only factory function exposed to the higher level was the `createService()` function (it is the only export of `src/factory/index.js`. All other factories are used internally by the implementation for `createService()` but should not be called directly by any modules in other directories. On a related note, `src/factory/util/*.js` leaked out [in a few places](https://github.com/patrick-steele-idem/ghcrawler/blob/b2b3cafa85167d9c0b43d2efecf3d421f7882836/src/options.js#L8) and I meant to go back and export that from `src/factory/index.js` (I'll make that change).